lib/beaker/options/presets.rb

```diff
@@ -181,6 +181,7 @@ def presets
           :ec2_yaml               => 'config/image_templates/ec2.yaml',
           :help                   => false,
           :collect_perf_data      => 'none',
+          :aws_keyname_modifier   => '',
```

Comment:
Why include anything external at all? Our goal isn't to add metadata, or is it? Milliseconds + random string creates a massive reduction in collision chance, probably enough for our needs.